controller/app/util/RequestInterceptor.java

```diff
@@ -43,46 +45,6 @@ private RequestInterceptor() {}
 
     // ---------------------------
     short var = 1;
-    apiHeaderIgnoreMap.put("/v1/user/create", var);
```

Comment:
@BharathwajShankar Why have we remove all these lines? These doesnot require user auth token in any env. so pls add these back

Reply:
added back the lines.